Updating collections, adding user and subseries guides

nichwall opened the pull request:
Updated language from v2.2.0 to v2.2.20 so things make a bit more sense. Also added two new guides to give a framework for other guides.

advplyr commented on content/guides/2.library_creation.md:
```diff
@@ -7,13 +7,14 @@ fullpath: /guides/library_creation
+# Setting up a library
 
-Libraries can contain either books or podcasts. To create a library file in ABS:
+ABS supports two kinds of libraries: book and podcast. The podcast library can automatically download new episodes from an RSS feed, but this is covered in another guide [insert link].
```
Is this a guide you are working on?

nichwall replied:
Yes, I'm working on a podcast guide and thought it fit better there (haven't gotten very far with it yet).

Could be written as "... RSS feed, covered more in this guide (link)" once I finish the guide, and leave that last clause out for now.

advplyr commented on content/guides/3.users.md:
```diff
@@ -0,0 +1,20 @@
+---
+
+# Introduction
+A distinct user should be created for every individual who will be using the server. There is currently no functional difference between a "Guest" and "User" type.
```
There is a small difference. Guest accounts cannot change their password.

advplyr commented on content/guides/5.collections.md:
```diff
@@ -0,0 +1,56 @@
+- Listening progress is tracked by user (not shared listening state between multiple users)
+- Have a description field
+- Can be reordered
+- Can contain podcast episodes
```
Currently collections can only contain books. We may expand this in the future but so far there hasn't been a need.

nichwall replied:
Yep, you're correct. I mixed up which list was which.

advplyr commented on content/guides/5.collections.md:
```diff
@@ -0,0 +1,56 @@
+
+![Initial collection](/guides/collections/1_no_collections.png)
+
+For a book library, a collection or playlist is created from an individual book. You can do this either from the library view or when viewing a specific book's details.
+Collections and playlists are created from book view. To do this, you can either do this from the library view or when viewing a specific book's details. In the second gif, the button has moved to the dropdown list of 3 dots in the same location.
```
I think there is some redundancy here?

nichwall replied:
Oh, yep. I didn't notice that with my multiple drafts of sentences getting pasted together. I'll clear that up.

advplyr commented:
This is excellent, thanks for making these.

nichwall commented:
I addressed your comments in this new commit, and added the "roadmap items" section to the end of `5.collections.md`, but there is currently only one item in that list right now. I'm sure there's other things that are on the roadmap, but I haven't been following the Discord or new issues super closely for the past two weeks.

Thanks for taking the time to review it!

advplyr commented on content/guides/5.collections.md:
```diff
@@ -0,0 +1,60 @@
+---
+
+# Introduction
+This page describes the similarities and differences between a collection and playlist. Collections and podcasts are another way to organize individual books and podcast episodes (not podcasts).
```
```suggestion
This page describes the similarities and differences between a collection and playlist. Collections and playlists are another way to organize individual books and podcast episodes (not podcasts).
```

advplyr commented:
Great! Thanks!
